src/simplification/SimplifyEnums.ts

```diff
@@ -9,56 +9,84 @@ type Output = any[] | undefined;
  */
 export default function simplifyEnums(schema: Schema | boolean, seenSchemas: Map<any, Output> = new Map()): Output {
   if (typeof schema !== 'boolean') {
```

Comment:
Also I think that we should go with this approach with handling boolean schemas:

```ts
if (typeof schema === 'boolean') {
  return udnefined;
}

// ...rest of logic
```

For me this is better to read and understand. Of course I can do that in follow up PR after yours.

Reply:
If we want consistency I think we should do that in another PR yes 😄